tests/test_exposition.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from pytheus.exposition import generate_metrics
+from pytheus.metrics import create_counter
+from pytheus.registry import REGISTRY, CollectorRegistry
+
+
+class TestExposition:
+
+    def setup_counters(self):
```

Comment:
this could be an autoused fixture if used by all the tests in this class but it's good like this!

Reply:
I need it this way so I can intialize it after tweaking the registry otherwise it will run before that.